ravenframework/Simulation.py

```diff
@@ -656,6 +658,8 @@ def __readRunInfo(self, xmlNode, runInfoSkip, xmlFilename):
         self.runInfoDict['internalParallel'] = utils.interpretBoolean(element.text)
         dashboard = element.attrib.get("dashboard",'False')
         self.runInfoDict['includeDashboard'  ] = utils.interpretBoolean(dashboard)
+      elif element.tag == 'parallelMethod':
+        self.runInfoDict['parallelMethod'] = ParallelLibEnum.__members__[element.text.strip().lower()]
```

Comment:
If the element.text is not in the ParalleLibEnum, the code will crash, but the error message may not helpful. Could you raise a meaningful error here?

Reply:
Yes, example error now:
```
OSError: Unknown parallel method: raynot, supported methods are ['dask', 'ray', 'pp', 'shared', 'distributed']
```